enterprise/api/v1/serializers.py

```diff
@@ -161,12 +161,10 @@ def save(self):  # pylint: disable=arguments-differ
         Save the model with the found EnterpriseCustomerUser.
         """
         course_id = self.validated_data['course_id']
-        consent_granted = self.validated_data['consent_granted']
 
         models.EnterpriseCourseEnrollment.objects.get_or_create(
             enterprise_customer_user=self.enterprise_customer_user,
             course_id=course_id,
-            consent_granted=consent_granted,
         )
```

Comment:
Do we still need to set the `consent_granted` flag here though if it was passed?

Reply:
I am guessing no because we aren't using this field anymore to indicate consent, so I think it should always be written as NULL or ideally removed altogether from the model. That was the intent I gleaned from the migration to the new consent architecture, but I could be wrong. Otherwise, we should change this call to update_or_create to allow updates to consent_granted which fail when the record exists for a user/course.

Reply:
Makes sense. Do we have a Jira ticket for removing the consent_granted field from the EnterpriseCourseEnrollment record in the backlog? @haikuginger

Reply:
The intent of the migration to the new consent architecture included deprecating reliance on consent through the enrollment model, but I had spoken with @haikuginger about completely removing this and related to-be-deprecated consent features, but it was decided that that should be considered in a later story.

Reply:
Yeah, to my knowledge, we do not have the needed followup story.